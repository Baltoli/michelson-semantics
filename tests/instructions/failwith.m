{ code { CDR ;  
         NIL operation ;
         PAIR ;
         FAILWITH 1
       }; 
storage unit;
parameter unit; } Unit Unit
