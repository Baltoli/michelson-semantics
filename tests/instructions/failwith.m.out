<generatedTop>
  <k>
    Aborted ( 1 )
  </k>
  <stack>
    FailedStack
  </stack>
</generatedTop>
